server/pages/index.js

```diff
@@ -369,8 +371,10 @@ module.exports = function() {
 				const pathRegex = utils.pathToRegExp( path );
 
 				app.get( pathRegex, function( req, res, next ) {
+					req.context = Object.assign( {}, req.context, { sectionName: section.name } );
```

Comment:
Why is this outside of the `code-splitting` conditional? If code splitting is disabled, there's just one bundle.